bee-common/bee-packable-derive/tests/pass/verify_with.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(unused_imports)]
+
+use bee_packable::{
+    error::UnpackErrorExt,
+    error::{UnknownTagError, UnpackError},
+    packer::Packer,
+    unpacker::Unpacker,
+    Packable,
+};
+
+use core::convert::Infallible;
+
+#[derive(Debug)]
+pub struct PickyError(u8);
+
+impl From<Infallible> for PickyError {
+    fn from(err: Infallible) -> Self {
+        match err {}
+    }
+}
+
+fn verify_value(&value: &u8) -> Result<(), PickyError> {
+    if value == 42 {
+        Ok(())
+    } else {
+        Err(PickyError(value))
+    }
+}
+
+#[derive(Packable)]
+#[packable(unpack_error = PickyError)]
+pub struct Picky(
+    #[packable(verify_with = verify_value)]
```

Comment:
Can we have multiple `verify_with` or should we always combine them all in one ? Just curious, not a request.

Reply:
only one is used but I'm not sure which one.